test/unit/sync.coffee

```diff
@@ -217,6 +217,8 @@ describe "Sync", ->
 
 
     describe 'applied', ->
+        @timeout 5000
```

Comment:
ok but I change the default timeout to 10s in the [`test/mocha.opts`](test/mocha.opts)